ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly-4.19-upgrade-from-stable-4.19.yaml

```diff
@@ -277,6 +311,41 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-ovn
+- as: aws-ipi-wavelength-byo-subnet-role-sts-mini-perm-f60
```

Comment:
We should update the existing [one](https://github.com/openshift/release/blob/5b0a4dc4a5f67b05e23899d76d3aec729dba39e6/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly-4.19-upgrade-from-stable-4.19.yaml#L361), rather than add a new one, right ?